ci/schema/duckdb.sql

```diff
@@ -45,11 +45,14 @@ CREATE OR REPLACE TABLE awards_players (
     notes TEXT
 );
 
+CREATE TYPE enum_t AS ENUM ('alice', 'bob', 'carol');
```

Comment:
```suggestion
DROP TYPE IF EXISTS enum_t CASCADE;
CREATE TYPE enum_t AS ENUM ('alice', 'bob', 'carol');
```

This needs to be recreated on every test run, so the type needs to be dropped if exists.